### ISSUE TITLE
Why aren't my custom headers working?

### ISSUE THREAD

**Issue by alexwlchan** [alexwlchan/alexwlchan.net#803]
I set custom headers in my `netlify.toml` file, but they aren't showing up when I scan the site: https://securityheaders.com/?q=https%3A%2F%2Falexwlchan.net%2F&followRedirects=on

What's up with that?

**Comment by alexwlchan** [comment]
This includes all my caching headers!

**Comment by alexwlchan** [comment]
Also broken on books.alexwlchan.net 🤔

**Comment by alexwlchan** [comment]
I thought this might be an issue in my `netlify.toml`, so I started doing some experiments where I commented parts of it out. No joy.

Then I tried doing a local deployment with the Netlify CLI, and my custom headers appeared!

Hmm, so it's something to do with the way I deploy my Netlify site? I currently use [nwtgck/actions-netlify](https://github.com/nwtgck/actions-netlify) and reading the README I found this config option:

> `netlify-config-path`: `./netlify.toml` Path to `netlify.toml` (default: undefined)

I think that default is weird, but oh well, it is what it is.

Because it's undefined by default, it's not seeing my custom headers in `netlify.toml` and not deploying them. Adding that config option seems to have fixed it.

I considered setting up a GitHub Actions script to check my security headers are deployed correctly, but this has only broken once that I remember so it's not worth the effort. If it becomes a persistent issue, maybe.

Closing as done.
